### DISCUSSION TITLE
Accessing "real time" java RAM usage in a container

### DISCUSSION THREAD

**Discussion by kylerhenry** [itzg/docker-minecraft-server#1717] [Q&A] [answered]
Hey, is there a way to monitor the actual usage of the max ram within the container? Obviously there is `docker stats`, but that number uses the allocated amount for java, not the usage within the JVM.

**Comment by itzg** [comment, marked as answer]
The typical monitoring tools use JMX which can be enabled as described here https://github.com/itzg/docker-minecraft-server#enable-remote-jmx-for-profiling
